grammars/php.cson

```diff
@@ -728,11 +728,11 @@
     'name': 'keyword.operator.bitwise.php'
   }
   {
-    'begin': '(?i)\\b(instanceof)\\s+(?=[\\\\$a-z_])'
+    'begin': '(?i)\\b(instanceof)\\s+(?=[\\\\$a-zA-Z_])'
```

Comment:
Modifier in front is already marks this expression case insensitive, so this change should not be needed.

Reply:
It does fix the issue though, hm...

Reply:
Oh, only the second line is needed

Reply:
Yeah, I realized that it could be read like that, sorry about that.